subsys/mgmt/mcumgr/lib/cmd/fs_mgmt/src/fs_mgmt.c

```diff
@@ -5,30 +5,27 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <assert.h>
-#include <zephyr/kernel.h>
```

Comment:
Is this really not needed? Are the CONFIG_* defines still present?

Reply:
I think this is taken care by this https://github.com/zephyrproject-rtos/zephyr/blob/main/CMakeLists.txt#L269-L270